Update Registrar Section (#5746)
w3f/polkadot-wiki: docs/learn/learn-identity.md

Unless no additional information is available here, you must reach out to specific registrars
individually if you want to be judged by those.

:::info Registrar 0 decommissioned on April 1st, 2024

From the 1st of April 2024 onwards, Registrar 0 will still exist on-chain but will not accept any

> new judgment requests. The registrar fees is set to a substantial amount to dissuade identity judgement

Reviewer: ```suggestion
new judgment requests. The registrar fees are set to a substantial amount to dissuade identity judgment
```

Reviewer 2: The section title itself is "Judgement" :)

The Wiki doc stuck to UK english with this word.
